libcst/_parser/conversions/expression.py

```diff
@@ -861,7 +878,7 @@ def convert_atom_basic(
     elif child.type.name == "NUMBER":
         # We must determine what type of number it is since we split node
         # types up this way.
-        if re.fullmatch(INTNUMBER_RE, child.string):
+        if re.fullmatch(INTNUMBER_RE + "[Ll]?", child.string):
```

Comment:
Shouldn't this be version-specific?

Reply:
I went ahead and made Py2Integer and split this out.